src/main/java/com/google/cloud/tools/appengine/cloudsdk/serialization/CloudSdkVersion.java

```diff
@@ -142,7 +155,8 @@ public int compareTo(CloudSdkVersion other) {
 
   @Override
   public int hashCode() {
-    return Objects.hash(majorVersion, minorVerion, patchVersion, preRelease, buildIdentifier);
+    return Objects.hash(
```

Comment:
Hmm, should we be hashing `version`?  Consider "0.0.1" vs "0.0.01": we previously considered them the same.

Reply:
I was thinking of a possibility in the future another version string such as "unknown" or a commit SHA hash could be associated with the value -1. But considering that `new CloudSdkVersion("0.0.1").compareTo(new CloudSdkVersion("0.0.01")) == true`, it makes sense to drop `version` from computing the hash.